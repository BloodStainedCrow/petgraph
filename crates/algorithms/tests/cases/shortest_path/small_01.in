version https://git-lfs.github.com/spec/v1
oid sha256:a4f286748128eddb8f61856d21a22d75c1e3f23bd24120e064df4a98493d4917
size 863
