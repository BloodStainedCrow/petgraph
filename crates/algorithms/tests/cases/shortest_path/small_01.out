version https://git-lfs.github.com/spec/v1
oid sha256:17c0c7e5885e153991b09bb5736b0b5a277838c7dfb2c3f5a0ad646c4838f6ef
size 45
